tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "es2020",
     "lib": ["es2020", "esnext"],
-    "module": "es2020",
+    "module": "Node16",
```

Comment:
:question: Why was this necessary?

Reply:
VS-code told me and Laurens it was :shrug: